src/test/java/teammates/ui/webapi/CreateAccountActionTest.java

```diff
@@ -39,27 +39,34 @@ protected void testExecute() {
         String institute = "TEAMMATES Test Institute 1";
 
         ______TS("Not enough parameters");
-        AccountCreateRequest badRequest = buildCreateRequest(null, institute, email);
-
-        try {
-            getAction(badRequest).execute();
-        } catch (InvalidHttpRequestBodyException e) {
-            assertEquals("name cannot be null", e.getMessage());
-        }
-
-        badRequest = buildCreateRequest(name, null, email);
-        try {
-            getAction(badRequest).execute();
-        } catch (InvalidHttpRequestBodyException e) {
-            assertEquals("institute cannot be null", e.getMessage());
-        }
-
-        badRequest = buildCreateRequest(name, institute, null);
-        try {
-            getAction(badRequest).execute();
-        } catch (InvalidHttpRequestBodyException e) {
-            assertEquals("email cannot be null", e.getMessage());
-        }
+
+        Exception exception;
+        String expectedError;
+        String actualError;
+
+        exception = assertThrows(InvalidHttpRequestBodyException.class,
+                () -> getAction(buildCreateRequest(null, institute, email)).execute());
+
+        expectedError = "name cannot be null";
+        actualError = exception.getMessage();
+
+        assertTrue(actualError.contains(expectedError));
```

Comment:
- Any reason for `contains` while it could be a strict equality check?
- Nitpicking, but this entire construct is too verbose. You can shorten to the following without losing any readability:
   ```java
   exception = assertThrows(InvalidHttpRequestBodyException.class,
           () -> getAction(buildCreateRequest(name, null, email)).execute());
   assertEquals("name cannot be null", exception.getMessage());
   ```

Reply:
Thank you for the feedback, appreciate the much neater and shorter solution! I have modified the code following your advice!